tools/interactive/interactivetool_pangeo_notebook.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="interactive_tool_pangeo_notebook" tool_type="interactive" name="Interactive Pangeo Notebook" version="1c0f66b">
```

Comment:
```suggestion
<tool id="interactive_tool_pangeo_notebook" tool_type="interactive" name="Interactive Pangeo Notebook" version="22.01" profile="22.01">
```
Hashes as versions as opposed to major-minor-patch version will make the tool unsortable by version. I don't know if there's maybe a better version you can use ?